mash_client/cli_utils.py

```diff
@@ -96,6 +96,25 @@ def get_config(cli_context):
     return data
 
 
+def update_config(cli_context, key, value):
+    """
+    Update a key in the current config profile.
+    """
+    config_dir = cli_context['config_dir'] or default_config_dir
+    profile = cli_context['profile'] or default_profile
+    config_path = config_dir + profile + '.yaml'
```

Comment:
Should be careful in how this is concatenated since it's valid for a user to leave off the trailing slash for config_dir. Should probably do an os path join with dir and {profile}.yaml.

Reply:
That being said I realize the usage that already exists throughout the client has this same problem. So I can submit a fix that's separate to clean that all up and this can be disregarded for the current PR.

Reply:
Created https://github.com/SUSE-Enceladus/mash-client/issues/56 for tracking this.